esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -569,8 +571,8 @@ define([
             registry.byId(this.id + "Recover").set("disabled", isArchived || !this.wu.isComplete() || this.wu.isDeleted());
             registry.byId(this.id + "Publish").set("disabled", isArchived || !this.wu.isComplete() || this.wu.isDeleted());
             registry.byId(this.id + "ZapReport").set("disabled", this.wu.isDeleted());
-            registry.byId(this.id + "Reschedule").set("disabled", !this.wu.isComplete() || this.wu.isFailed());
-            registry.byId(this.id + "Deschedule").set("disabled", this.wu.isComplete() || this.wu.isFailed());
+            registry.byId(this.id + "Reschedule").set("disabled", !this.wu.isAbleToDeschedule());
```

Comment:
If it's calling a function called "isAbleToDeschedule" then surely it's the deschedule button that should be enabled/disabled accordingly. Either the logic is wrong or the function name is confusing!